srcs/layouts/latn_azerty_fr.xml

```diff
@@ -1,39 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Adapted from standard NF Z71-300 -->
+<!-- Adapted from Azerty Amélioré standard -->
 <!-- https://norme-azerty.fr/ -->
 <keyboard name="AZERTY (Français)" script="latin">
   <row>
-    <key key0="a" key2="1" key4="loc esc"/>
-    <key key0="z" key2="2" key3="&amp;" key4="~"/>
-    <key key0="e" key2="3" key3="&quot;" key4="\#"/>
-    <key key0="r" key2="4" key3="'"/>
-    <key key0="t" key2="5" key3="(" key4=")"/>
-    <key key0="y" key2="6" key3="-" key4="|"/>
-    <key key0="u" key2="7" key3="à" key4="`"/>
-    <key key0="i" key2="8" key3="_" key4="\\"/>
-    <key key0="o" key2="9" key3="\@"/>
-    <key key0="p" key2="0"/>
+    <key key0="a" key2="1" key4="à"/>
+    <key key0="z" key1="accent_aigu" key2="2" key3="é" key4="½"/>
+    <key key0="e" key1="accent_grave" key2="3" key3="è" key4="æ"/>
+    <key key0="r" key1="accent_circonflexe" key2="4" key3="ê" key4="&amp;"/>
+    <key key0="t" key1="accent_trema" key2="5" key3="(" key4=")"/>
+    <key key0="y" key1="accent_macron" key2="6" key3="[" key4="]"/>
+    <key key0="u" key2="7" key3="‘" key4="’"/>
+    <key key0="i" key1="—" key2="8" key3="–" key4="*"/>
+    <key key0="o" key1="œ" key2="9" key3="«" key4="»"/>
+    <key key0="p" key1="&quot;" key2="0" key3="'"/>
   </row>
   <row>
-    <key key0="q" key2="loc tab"/>
-    <key key0="s" key3="loc ß"/>
-    <key key0="d" key1="loc accent_grave" key2="é" key3="loc accent_aigu"/>
-    <key key0="f" key3="{" key4="}"/>
-    <key key0="g" key3="[" key4="]"/>
-    <key key0="h" key3="=" key4="+"/>
-    <key key0="j" key1="loc accent_trema" key2="loc accent_circonflexe" key3="^"/>
-    <key key0="k" key1="è" key2="€" key3="$"/>
-    <key key0="l" key2="%"/>
-    <key key0="m" key3="*"/>
+    <key key0="q" key2="loc tab" key4="loc esc"/>
+    <key key0="s" key1="\@" key2="\#" key3="loc ß" key4="§"/>
+    <key key0="d" key1="`" key2="€" key3="^" key4="$"/>
+    <key key0="f" key3="~" key4="¤"/>
+    <key key0="g" key3="{" key4="}"/>
+    <key key0="h"/>
+    <key key0="j" key1="ù" key3="“" key4="”"/>
```

Comment:
Symbols like ÷×… and quotes are available with the Fn key. I think that should be sufficient ?